ocd_backend/sources/ori.go.yaml

```diff
@@ -109,11 +109,12 @@ ori.go:
     almanak_id: 25660
     <<: *go
 
-  molenwaard:
-    key: molenwaard
-    base_url: "https://raad.gemeentemolenwaard.nl/api"
-    almanak_id: 27268
-    <<: *go
+  # Removed. Merged with Molenlanden on 1-1-2019
```

Comment:
This can be completely removed instead of commented since it isn't coming back.

Reply:
Resolved in a5f1d677ed97690b02323a0fe3d5b549b062f275